### DISCUSSION TITLE
AssertionError: The length of palette should not be less than `num_classes`.

### DISCUSSION THREAD

**Discussion by oceanechy** [open-mmlab/mmyolo#348] [Q&A]
自定义数据集进行训练后，在进行验证单张图片时出现此类错误，无法对图片进行画图，这里我的类别数是15个，我对metainfo中的'PALETTE'添加至15个类别还是不行，能帮帮我吗，十分谢谢，下面是我在metainfo中更改的部分:
metainfo = {
    'CLASSES': ('small-vehicle', 'large-vehicle', 'plane', 'storage-tank', 'ship', 
        'harbor', 'ground-track-field', 'soccer-ball-field', 'tennis-court', 
        'swimming-pool', 'baseball-diamond', 'roundabout', 'basketball-court',
         'bridge', 'helicopter'),
    'PALETTE': [
       (220, 20, 60),(220, 20, 70),(220, 20, 80),(220, 20, 90),(220, 20, 100),(220, 20, 110),(220, 20, 120),(220, 20, 130),(220, 20, 140),(220, 20, 150),(220, 20, 160),(220, 20, 170),(220, 20, 180),(220, 20, 190),(220, 20, 200)#(220, 20, 210),(220, 20, 220),
    ]
}

**Comment by RangeKing** [comment]
@Oceanchy6，训练的时候 PALETTE 不是15个类别？
